packages/samples/specs/resource-manager/tenantResource/main.tsp

```diff
@@ -180,6 +180,7 @@ model FlangeProperties {
   ...DefaultProvisioningStateProperty;
 }
 
+#suppress "deprecated" "Should replace TenantResourceDelete in the future"
```

Comment:
@markcowl I was able to update the other samples to simply use `ArmResourceDeleteWithoutOkAsync` but this is an interface template of a template of a template, etc. so I just suppressed the deprecation warning...